free-3.md

```diff
@@ -10,41 +10,98 @@ _What need is met by your idea? Who are you helping? What is that person's
 
+_The user would have to load their cards into the program and tell the language
+which version of the rules it ants to use. Programming looks like changing the
+cards and rules to be able to play a game against the computer. This is the
+right way to interact with the problem domain because there are so many cards
+for any given game that it would be ridiculous to include all of them.
+Additionally, card developers release new cards very often to continue to make
+a profit, so a designer could not account for every single card. However, each
+card has a certain template that it follows. So, once the language learns
+that template, it will be able to understand any card that the user programs
+into it._
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+_When a program runs, it will give the user a way to play the game with the
+computer. The errors that might occur would be incorrect cards. For instance,
+if a card did not have a certain field or the field was filled in with the
+wrong type, the program should not accept it as a card. So, it would tell
+the user which card was wrong and why and encourage the user to try the 
+program again with a fix. Additionally, if a user tried to make a move that
+was not allowed by the rules, the program should reset to the previous
+position and tell the user that the previous move was not allowed._
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+_It should be easy to make new cards for a given game. It shoud be difficult,
+but possible to make a completely new game because that would involve writing
+all of the rules into place and describing how a card is set up. It should be
+impossible to run a program in this language that just prints "Hello World."_
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+_There are currently programs that will let you play these games online with
+other people (http://magic.wizards.com/en/content/magic-online-products-game-info
+or http://www.pokemon.com/us/pokemon-tcg/play-online/). I admire that most of
+them have info for new players and tutorials to help them get started. I wish
+that there were more starter sets for players so that they would not have to
+try to craft their own decks and ways for players to make their own cards, which
+is not a part of these languages because they are owned by the company that
+makes these games. I also wish that there were more variablity if people wanted
```

Comment:
A little besides the point, but Hearthstone is an online card game that is free. The game pairs you with players in your level, so it tries to help beginners by having them play other beginners. Hearthstone is a DSL, to the extent to which card games are DSLs, that addresses the same needs other than complete customization.